### DISCUSSION TITLE
CWG goals for 2024

### DISCUSSION THREAD

**Discussion by russelltrow** [Green-Software-Foundation/community_wg#103] [General]
As we move into the new year this is a great opportunity for us to start discussing what we would like to see the Community Working Group focus on.

There are currently a number of active projects in the WG:
* Green Maturity Matrix
* Green Software Patterns
* Green Software Practitioner course

There is also a proposal in the pipeline for a new project:
* [Higher Education strategy](https://github.com/Green-Software-Foundation/community-wg/issues/89)

In the [2024-01-17 - Green Software Community Working Group](https://github.com/Green-Software-Foundation/community-wg/issues/100) meeting we started to identify problems that the GSF & Member Organisations were currently facing and might be great areas for the CWG to explore further. **Below is a list of the initial areas and ideas captured in that meeting.**

**Comment by russelltrow** [comment]
## How might we scale the GSF's engagement with Member Organisations?

* We only have 1 Namrata
* We lack a CRM tool to make this process easier
* Namrata wants to raise this issue with the Steering Committee

### Initial Questions
* How are other foundations scaling member engagement? Assume they are bringing in 3rd parties
* How are we currently tracking our engagement with Member Orgs?
* What is the GSF strategy on this?

**Comment by russelltrow** [comment]
## How might we build bridges between GSF organisations and support knowledge sharing?

* A value of the GSF is the collective knowledge of all the Member Organisations, how can we bring them together in a scalable way?

### Initial Thoughts

* “We could ask the member organisation to provide you with their regional contact that we can publish in the GSF website. That would allow the regional leaders for example to connect locally”
* “This feeds into the idea that Sarah has come up with - where we create roundtables for companies within industries”

### Initial Questions

* "Does the GSF Organization Lead get informed when a new member signs up or registeres in a working group or project by the GSF or automatically?"

**Comment by russelltrow** [comment]
## How might we make the topic of Green IT less overwhelming?

* “spot on Felipe - the topic of sustainability and even Green IT is so so wide and broad. Overwhelming is the right word!”

## Initial Questions

* "What approaches do coaching companies take to help people flourish?"
* "How can we help to foster a distributed Learning Organisation where people feel safe to experiment?"

**Comment by russelltrow** [comment]
## How might we support internal networking within Member Organisations?

* Namrata keeps being asked to help make connections within Member Orgs where a stakeholder isn't connected to another stakeholder in the same org who are both active in green software.

### Initial Thoughts

* “I personally think the internal org comms connections should be left with the org. The value GSF can add is the topics that are coming up across organisations and create a log of all those. That's when we can connect orgs together”
* “Ask the organisations to take responsibility for what happens within their own org. Offload to the respective organisation””
* “I think it is best to introduce folks to a key stakeholder within their organization and we can continue to discuss in this working group if we see unique challenges.”

**Comment by russelltrow** [comment]
## How might we support internal green advocacy at scale?

* Provide organisations the tools they need to drive adoption. Focus on enablement materials that can be used by anyone
  * Resource pack for internal advocacy, 
    * PDF of small changes e.g. Forming a green team
    * Tooling to use
    * courses to do 
    * books to read 

## Initial Ideas

* Combination of Training and Use Cases
* “Carbon Hacks” within organisations
* Matrix Maturity as the cornerstone

**Comment by russelltrow** [comment]
## Idea: Cohort based training within organisations

* @aecurrie raised the idea of cohort based training to build groups of evangelists within organisations.

**Comment by russelltrow** [comment]
## Idea: “Carbon Hacks within Member Orgs”

* @PindyBhullar raised the idea of “Carbon Hacks” within organisations
* Make it easier to get comfortable with green software and deliver something valueable at the end
* Help practitioners put knowledge gained through any training into practise quickly

**Comment by russelltrow** [comment]
## Idea: GSF Member Org Conference

* Internal conference to promote knowledge-sharing within GSF Members
* Create opportunities for knowledge-sharing
* Platform to speak with other members and identify common challenges 
* Bi-annually?
* Hybrid?

**Comment by russelltrow** [comment]
## Idea: GSF internal Town Hall / All Hands

* For GSF members
* Quarterly event
* Members come and tell their stories

**Comment by NAMRATA-WOKE** [comment]
## Idea: Focus Series

Hold a 'focus series' for the next ~4 meetings, where each CWG meeting is meant to deep-dive into a particular interest area. 

Reason: Those really interested and can add value, experience, perspectives, will be encouraged to participate
Outcome: List of actions and SWOT analysis of opportunity

**Comment by russelltrow** [comment]
## Idea: Higher Education specific GSF onboarding 

Higher Education GSF members are struggling to engage with the GSF. This is due to:

* Significant usage of terminology that is not familiar to them
* Projects which move at a much faster pace than they are used to
* Their aims are different to commercial members
* They are interested in what it means to design green software rather than projects relevant to later in the SDLC

**How might we provide them a better onboarding experience?**

@tholewis might this be relevant to the HE project?

**Reply by tholewis** [reply]
@russelltrow - These are great insights and will incorporate into the plan and what we are intending to accomplish. I think onboarding experience is absolutely something we want to consider and think we made some initial good insights into it from the Green Patterns workshop.
